test/util/scripts/reconciliationTransfer/01populateAdminTestData.sh

```diff
@@ -118,7 +118,7 @@ do
     -H 'Content-Type: application/json' \
     -d '{
     \"type\": \"FSPIOP_CALLBACK_URL_TRANSFER_ERROR\",
-    \"value\": \"http://localhost:1080/${FSP}/transfers/{{transferId}}/error\"
+    \"value\": \"http://localhost:1080/${FSP}/transfers/{{id}}/error\"
```

Comment:
I am somewhat concerned about this change. The reason being that keep in mind that existing DBs are already using templates with `transferId`. Would we not have to include scripts to update all existing records to use `id` instead of `treansferId`?

Reply:
Restored back for regular transfers